sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/microsoft/opentelemetry/exporter/azuremonitor/export/__init__.py

```diff
@@ -47,9 +48,26 @@ def __init__(self, **options):
         default_storage_path = os.path.join(
             tempfile.gettempdir(), TEMPDIR_PREFIX + temp_suffix
         )
-        retry_policy = RetryPolicy(timeout=self._timeout)
+
+        config = AzureMonitorClientConfiguration(parsed_connection_string.endpoint, **kwargs)
```

Comment:
Shouldn't `self._timeout` be passed into **kwargs so `RetryPolicy` will pick it up?

Reply:
This is being addressed in this PR #15396 

We will be using connection_timeout config for it, I will merge the changes